packages/builder/src/pages/builder/portal/users/users/[userId].svelte

```diff
@@ -119,7 +120,11 @@
         name: app.name,
         devId: app.devId,
         icon: app.icon,
-        role: privileged ? Constants.Roles.ADMIN : roles[prodAppId],
+        role: privileged
+          ? Constants.Roles.ADMIN
+          : user?.builder?.apps.includes(prodAppId)
+          ? Constants.Roles.CREATOR
+          : roles[prodAppId],
```

Comment:
NAB: don't need to change this, but in general I think these multi condition ternary statements are very hard to read and understand. In my opinion, if it's anything more than a basic ternary statement then it should be a dedicated function.